_posts/2013-04-06-developing-addons-and-blueprints.md

```diff
@@ -260,14 +260,24 @@ import Ember from 'ember';
 
 export default Ember.Component.extend({
   tagName: 'button',
-
-  setupXbutton: Ember.on('didInsertElement', function() {
+  
+  didInsertElement() {
```

Comment:
Confirm, you should avoid using the lifecycle events and instead override the hooks.